src/lib/requests.ts

```diff
@@ -30,17 +30,19 @@
  --------------
  ******/
 /* istanbul ignore file */
-// Testing will be covered in PR #11
+// Testing will be covered in #354
 
 import { config } from './config'
-// @ts-ignore
-import { Logger, ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import { ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import Logger from '@mojaloop/central-services-logger'
+// TODO: Switch to sdk-standard-components Logger once implemented
 
 // Config file to instantiate ThirdPartyRequest object
 const configRequest: BaseRequestConfigType = {
   dfspId: config.get('PARTICIPANT_ID') as string,
```

Comment:
why typecasting, if you would like to be sure you have a string in execution time use `.toString()` -> typecasting is only compilation time construct

Reply:
was typecasting to avoid a TS error
Even if i use .toString - same error is thrown - as TS cannot parse the type 
also @spikerheado1234  in your lib config file you've done the same - did you mean to do that? perhaps you should also be doing .toString because it seems if you don't you're getting returned an object?

Reply:
I am starting to love this: `dfspId: config.get('PARTICIPANT_ID').toString() as string`
maybe using https://www.npmjs.com/package/@types/convict would solve the problem?

Reply:
i tried but it didn't
so finally have reverted back to initial implementation